python_modules/dagster/dagster/_vendored/toposort/__init__.py

```diff
@@ -0,0 +1,40 @@
+## Vendored toposort module
+
+"""Vendored from https://gitlab.com/ericvsmith/toposort/-/blob/master/src/toposort.py?ref_type=heads on 2024-05-30.
```

Comment:
We should probably pin to a specific git commit or tag.

Reply:
agree with this - the 1.10 tag seems reasonable since its the most recent release